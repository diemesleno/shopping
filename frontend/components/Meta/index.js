import Head from 'next/head';

export default function Meta() {
  return (
    <Head>
      <meta name="viewport" content="width=device-width, initial-scale=1" />
      <meta charSet="utf8" />
      <link rel="shortcupt icon" href="/static/favicon.png" />
      <link rel="stylesheet" type="text/css" href="/static/nprogress.css" />
      <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet" />
      <title>gems</title>
    </Head>
  );
}