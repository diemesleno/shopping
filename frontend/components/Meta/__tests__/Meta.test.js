import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Meta from '../index';

describe('<Meta />', () => {
  it('renders', () => {
    const wrapper = shallow(<Meta />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
