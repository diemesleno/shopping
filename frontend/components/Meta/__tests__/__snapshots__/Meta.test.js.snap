// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Meta /> renders 1`] = `
<Head>
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <meta
    charSet="utf8"
  />
  <link
    href="/static/favicon.png"
    rel="shortcupt icon"
  />
  <link
    href="/static/nprogress.css"
    rel="stylesheet"
    type="text/css"
  />
  <link
    href="https://fonts.googleapis.com/css?family=Quicksand&display=swap"
    rel="stylesheet"
  />
  <link
    href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap"
    rel="stylesheet"
  />
  <title>
    gems
  </title>
</Head>
`;
